scss/6-components/_search.scss

```diff
@@ -33,6 +34,14 @@
     .cads-lang-cy &::after {
       content: 'Chwilio';
     }
+
+    &:active {
```

Comment:
This resets styles from the new default buttons. This is probably fine, but do wonder if it's preferable to special-case the search bar and inline just the required styles rather than unsetting things. Not a big deal.